src/sentry/api/endpoints/team_projects.py

```diff
@@ -164,6 +164,13 @@ def post(self, request: Request, team) -> Response:
 
         if not serializer.is_valid():
             return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+        if (
+            team.organization.flags.disable_member_project_creation
+            and not request.access.has_scope("org:write")
```

Comment:
This would disable creating new projects for even team admins that are normal org members, is this intended?